Sample Data/NetCleanProActiveSampleData.csv

```diff
@@ -0,0 +1,4 @@
+﻿"TimeGenerated [UTC]","domain_s","hasCollectedNearbyFiles_s","externalIP_s","nearbyFiles_md5s_s","Hostname_s","Identifier_g","type_s","version_s","foundTime_t [UTC]","detectionMethod_s","agentInformatonIdentifier_g","osVersion_s","machineName_s","microsoftCultureId_s","timeZoneId_s","microsoftGeoId_s","Agentversion_s","Agentidentifier_g","loggedOnUsers_s"

```

Comment:
@NCsteven, please check the format of sample data it should be added in array, please refer other [sample data](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Sample%20Data/Custom/AIShield_CL.json) for reference.
The file needs to be added at this [path](https://github.com/Azure/Azure-Sentinel/tree/master/Sample%20Data/Custom).